h2o-bindings/bin/custom/python/gen_targetencoder.py

```diff
@@ -29,3 +43,134 @@ def transform(self, frame, data_leakage_handling="None", noise=-1, seed=-1):
     __imports__="""import h2o""",
     __class__=class_extensions,
 )
+
+examples = dict(
+    blending="""
+>>> titanic = h2o.import_file("https://s3.amazonaws.com/h2o-public-test-data/smalldata/gbm_test/titanic.csv")
+>>> predictors = ["home.dest", "cabin", "embarked"]
+>>> response = "survived"
+>>> titanic["survived"] = titanic["survived"].asfactor()
+>>> fold_col = "kfold_column"
+>>> titanic[fold_col] = titanic.kfold_column(n_folds=5, seed=1234)
+>>> titanic_te = H2OTargetEncoderEstimator(k = 0.7,
+...                                        f = 0.3,
+...                                        data_leakage_handling= "None",
+...                                        blending = True)
+>>> titanic_te.train(x = predictors,
+...                  y = response,
+...                  training_frame = titanic)
+>>> titanic_te
+""",
+    data_leakage_handling="""
+>>> titanic = h2o.import_file("https://s3.amazonaws.com/h2o-public-test-data/smalldata/gbm_test/titanic.csv")
+>>> predictors = ["home.dest", "cabin", "embarked"]
+>>> response = "survived"
+>>> titanic["survived"] = titanic["survived"].asfactor()
+>>> fold_col = "kfold_column"
+>>> titanic[fold_col] = titanic.kfold_column(n_folds=5, seed=1234)
+>>> titanic_te = H2OTargetEncoderEstimator(k = 0.7,
+...                                        f = 0.3,
+...                                        data_leakage_handling= "k_fold",
```

Comment:
`data_leakage_handling="k_fold"`: this is the correct value.
Looks like there's an error in the documentation lines 10-12 then: 'KFold" -> 'k_fold' ...

Reply:
Just double checked that both spelling work, `k_fold` and `KFold`, but for documentation, maybe we should just stick to one. I suggest the `k_fold` case, as it is what the code generator is using for auto-generated doc.

Reply:
It's one of those cases when trying to be clever somewhere (in parsing values on backend) can create confusion somewhere else...

Reply:
Both versions work for each, but I'll switch it over to the `k_fold` case